torch/csrc/jit/runtime/register_special_ops.cpp

```diff
@@ -433,6 +459,39 @@ RegisterOperators reg({
         "aten::set_grad_enabled(bool val) -> ()",
         [](Stack* stack) { torch::GradMode::set_enabled(pop(stack).toBool()); },
         aliasAnalysisConservative()),
+    Operator(
+        "aten::getCurrentStream(int64_t val) -> __torch__.torch.classes.cuda.Stream",
+        [](Stack* stack) {
+          int64_t idx;
+          pop(stack, idx);
+          auto v = make_custom_class<torch::jit::CUDAStream>(idx);
+          push(stack, v);
+        },
+        aliasAnalysisFromSchema()),
+    Operator(
+        "aten::current_device() -> int",
+        [](Stack* stack) {
+          auto v = c10::cuda::current_device();
+          push(stack, v);
+        },
+        aliasAnalysisFromSchema()),
+    Operator(
+        "aten::set_device(int64_t val) -> ()",
+        [](Stack* stack) {
+          int64_t idx;
+          pop(stack, idx);
+          c10::cuda::set_device(idx);
+        },
+        aliasAnalysisFromSchema()),
+    Operator(
+        "aten::setCudaStream(torch.classes.cuda.Stream stream) -> ()",
+        [](Stack* stack) {
+          auto v = pop(stack);
+          auto stream = v.toCustomClass<torch::jit::CUDAStream>();
+          //auto bits = stream->pack();
+          c10::cuda::setCurrentCUDAStream(stream.get_stream);
```

Comment:
```suggestion
          c10::cuda::setCurrentCUDAStream(stream.get_stream());
```
You need to call the function here.